scene/resources/physics_material.cpp

```diff
@@ -43,9 +43,9 @@ void PhysicsMaterial::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_absorbent", "absorbent"), &PhysicsMaterial::set_absorbent);
 	ClassDB::bind_method(D_METHOD("is_absorbent"), &PhysicsMaterial::is_absorbent);
 
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "friction", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_friction", "get_friction");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "friction"), "set_friction", "get_friction");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "rough"), "set_rough", "is_rough");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "bounce", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_bounce", "get_bounce");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "bounce"), "set_bounce", "get_bounce");
```

Comment:
It makes sense to let users set higher values, but `,or_greater` should be added instead of removing the range entirely, since negative values are not useful (`rough` and `absorbent` are meant to be used instead and their effect is documented) and the interval `0..1` is still a good hint for common cases.

Reply:
This is purely a revert of #42203 to fix the regression. I think any new changes should be made in a separate PR, where they can be discussed, agreed (or left unmerged due to a lack of consensus) and, if necessary, reverted if they're wrong.

Reply:
I see no point in reverting the PR myself, since by far for most use cases it is useful and an improvement over what was in there. I would do a separate PR with _or_greater_ hint for the rare cases where you really want to go over 1.0.

Reply:
I've just opened #48434 to add `or_greater` hint instead of reverting the range.